<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d1_ProductItem</class>
 <widget class="QPushButton" name="d1_ProductItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#d1_ProductItem{
    background-color: rgb(255, 255, 255);
    border-radius:  12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="d1_widgetBachground" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="d1_lblImage">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="d1_lblName">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>40</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baloo Cyrillic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#d1_lblName{
	color: rgb(82, 18, 69);
}</string>
      </property>
      <property name="text">
       <string>Тест отображения имени</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="d1_lblNumber">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>117</width>
        <height>89</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baloo Cyrillic</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#d1_lblNumber{
	border-image: url(:/d1_roundYelBig.png);
	color: rgb(159, 39, 255);
}

#d1_lblNumber::disabled{
	border-image: url(:/d1_roundYelBigDisable.png);
	color: rgb(90, 90, 90);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="d1_lblPrice">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>158</width>
        <height>78</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baloo Cyrillic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#d1_lblPrice{
	color: rgb(255, 101, 0);
}</string>
      </property>
      <property name="text">
       <string>Цена</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="d1_lblStatus">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>76</width>
        <height>78</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="d1_resource/d1_resource.qrc">:/d1_hit.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="d1_resource/d1_resource.qrc"/>
 </resources>
 <connections/>
</ui>
